qiita_ware/util.py

```diff
@@ -108,113 +107,24 @@ def stats_from_df(df):
 
     Returns
     -------
-    dict
-        Dictionary object where the keys are the names of the metadata
-        categories and the keys are tuples where the first element is the name
+    dict of list of tuples
+        Dictionary object where the keys are the metadata categories
+        and the values are list of tuples where the first element is the name
```

Comment:
The next line should read something like "in category", should be "in the category".